dlp/metadata.js

```diff
@@ -15,102 +15,90 @@
 
 'use strict';
 
-const API_URL = 'https://dlp.googleapis.com/v2beta1';
-const requestPromise = require('request-promise');
-
-function listInfoTypes (authToken, category) {
+function listInfoTypes (category, languageCode) {
   // [START list_info_types]
-  // Your gcloud auth token.
-  // const authToken = 'YOUR_AUTH_TOKEN';
+  // Imports the Google Cloud Data Loss Prevention library
+  const DLP = require('@google-cloud/dlp');
+
+  // Instantiates a client
+  const dlp = DLP();
 
   // The category of info types to list.
   // const category = 'CATEGORY_TO_LIST';
 
-  // Construct REST request
-  const options = {
-    url: `${API_URL}/rootCategories/${category}/infoTypes`,
-    headers: {
-      'Authorization': `Bearer ${authToken}`,
-      'Content-Type': 'application/json'
-    },
-    json: true
-  };
+  // The BCP-47 language code to use, e.g. 'en-US'
+  // const languageCode = 'en-US';
 
-  // Run REST request
-  requestPromise.get(options)
-    .then((body) => {
-      console.log(body);
-    })
-    .catch((err) => {
-      console.log('Error in listInfoTypes:', err);
+  dlp.listInfoTypes({
+    category: category,
+    languageCode: languageCode
+  })
+  .then((body) => {
```

Comment:
Perhaps `apiResponse` would be a better name here - WDYT?

(Something like `listInfoTypesResponse` seems a little long-winded.)